.github/workflows/ci.yaml

```diff
@@ -585,7 +585,7 @@ jobs:
         run: |
           echo '```console' > "$GITHUB_STEP_SUMMARY"
           # Enable color output for pre-commit and remove it for the summary
-          SKIP=cargo-fmt,clippy,dev-generate-all pre-commit run --all-files --show-diff-on-failure --color=always | \
+          SKIP=cargo-fmt,clippy,dev-generate-all pre-commit run --all-files --show-diff-on-failure --color=always --hook-stage=manual | \
```

Comment:
Nit: It wasn't very clear to me what *manual* means. What would you think of renaming it to `CI`?

Reply:
Unfortunately this is out of my control 😆 it has to be one of git's recognised hook stages or `manual`: https://pre-commit.com/#confining-hooks-to-run-at-certain-stages

Reply:
I'll add a comment to this workflow file, though